website/src/install.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+set -e
+
+# This script installs the Git Town executable in the user's HOME directory.
+
+VERSION=7.6.0           # the version of Git Town to install
+DEST=$HOME/.local/bin   # the folder into which to install the Git Town executable
+TMP_DIR=.git-town-download  # temporary folder to use
+
+
+main() {
+  print_welcome
+
+  # verify the environment
+  need_cmd uname
+  need_cmd curl
+  need_cmd echo
+  OS="$(os_name)"
+  if [ "$OS" = "other" ]; then
+    err "Unsupported operating system, please install from source."
+  fi
+  CPU="$(cpu_name)"
+  if [ "$CPU" = "other" ]; then
+    err "Unsupported CPU architecture, please install from source."
+  fi
+  EXECUTABLE_FILENAME=$(executable_filename "$OS")
+  DEST_PATH=$DEST/$EXECUTABLE_FILENAME
+  rm "$DEST_PATH" > /dev/null 2>&1
+  ensure_no_other_git_town "$DEST_PATH"
+  echo "No other Git Town installation found, proceeding to install Git Town $VERSION for ${OS}_${CPU}."
+  echo
+
+  # download the executable
+  URL="$(download_url "$OS" "$CPU")"
+  tput dim
+  download_and_extract "$URL" "$OS" "$EXECUTABLE_FILENAME"
+  tput sgr0
+  echo
+  echo "I have successfully:"
+  echo "- downloaded $URL"
+  echo "- extracted the Git Town $VERSION binary to $DEST_PATH"
+
+  # unpack the archive
+  rm -rf "$TMP_DIR"
+  check_path "$EXECUTABLE_FILENAME"
+}
+
+print_welcome() {
+  echo "GIT TOWN INSTALLER SCRIPT"
+  echo "This installer is under development. Please report bugs at https://github.com/git-town/git-town/issues/new."
+  echo
+}
+
+
+# provides the name of the operating system in the format used by release assets
+os_name() {
+  case "$(uname -s)" in
+    Darwin*)  echo "macOS"   ;;
+    Linux*)   echo "linux"   ;;
+    MSYS*)    echo "windows" ;;
+    cygwin*)  echo "windows" ;;
+    *)        echo "other"
+  esac
+}
+
+# provides the CPU architecture name
+cpu_name() {
+  cpu_name=$(uname -m)
+  case $cpu_name in
+    x86_64 | x86-64 | x64 | amd64)  echo "intel_64" ;;
+    aarch64 | arm64)                echo "arm_64"   ;;
+    *)                              echo "other"
+  esac
+}
+
+# provides the URL from which to download the Git Town asset for the given OS and cpu type
+download_url() {
+  OS=$1
+  CPU=$2
+  EXT=tar.gz
+  if [ "$OS" = windows ]; then
+    # only Intel binaries for Windows right now
+    CPU=intel_64
+    EXT=zip
+  fi
+  if [ "$OS" = macOS ]; then
+    # only Intel binaries for macOS right now
```

Comment:
What is causing this? I have a new macbook that is arm64

Reply:
We just didn't get around to enabling this target platform in goreleaser, and didn't have a way to test it. More than happy to add this in!